run/tesla_api.py

```diff
@@ -107,6 +113,9 @@ def _rest_request(url, method=None, data=None):
         response = requests.post(url, headers=headers, data=data)
     else:
         raise ValueError('Unsupported Request Method: {}'.format(method))
+    if  'invalid bearer token' in response.text:
```

Comment:
this should be below the if on line 119 below, I think?
(otherwise it'll crash if response.text=None)

Reply:
Good catch, fixing!